newIDE/app/src/UI/SemiControlledAutoComplete.js

```diff
@@ -148,8 +148,9 @@ const handleChange = (
   if (option.type !== 'separator') {
     if (option.onClick) option.onClick();
     else {
-      if (props.onChoose) props.onChoose(option.value);
-      props.onChange(option.value);
+      props.onChoose
+        ? props.onChoose(option.value)
+        : props.onChange(option.value);
```

Comment:
We could call both the functions together, there is nothing wrong in doing that, but it doesn't make any sense. Only one of them selects the option (even when I called both of them) as far as I could understand :)